src/Private/Server.ps1

```diff
@@ -140,8 +140,12 @@ function Start-PodeInternalServer {
         # run running event hooks
         Invoke-PodeEvent -Type Running
 
+        # Check if the running version of Powershell is EOL
+        $null = Test-PodeVersionPwshEOL
```

Comment:
Do you want to check for EOL only or also for not tested versions?

Reply:
Yeah, I'd add `-ReportUntested` to it. I just did some quick testing locally as well, and I'm also thinking we move this line, and the line which write "Pode v2.10.0 (PID: X)" to the top of the `Start-PodeInternalServer` function - just so they are one of the very first messages output by Pode:

```powershell
function Start-PodeInternalServer {
    param(
        [Parameter()]
        $Request,

        [switch]
        $Browse
    )

    try {
        # Check if the running version of Powershell is EOL
        Write-PodeHost "Pode $(Get-PodeVersion) (PID: $($PID))" -ForegroundColor Cyan
        $null = Test-PodeVersionPwshEOL -ReportUntested

        ...
```

I also tweaked the version colour to Cyan 😉